### PR TITLE
Add HWP Mimetype icon

### PR DESCRIPTION
**Pull request by kuroehanako**

### CODE REVIEW

**Comment by Muqtxdir** [comment]
Hi @kuroehanako 

you do need to render the svgs

File: icons/src/fullcolor/default/mimetypes/applications-x-hwp.svg [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+         xml:space="preserve"
+         x="146.48828"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;line-height:0%;font-family:'DejaVu Sans';-inkscape-font-specification:'DejaVu Sans Bold';text-align:start;writing-mode:lr-tb;text-anchor:start;display:inline;fill:#000000;fill-opacity:1;stroke:none;enable-background:new"
+         inkscape:label="icon-name"
+         id="icon-name"><tspan
+           style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:18px;line-height:1.25;font-family:'DejaVu Sans';-inkscape-font-specification:'DejaVu Sans Bold'"
+           y="-8.2548828"
+           x="146.48828"
+           sodipodi:role="line"
+           id="tspan3937">x-office-document</tspan></text>
```
Comment (Muqtxdir) [inline icons/src/fullcolor/default/mimetypes/applications-x-hwp.svg:398]: The baseplate name should be changed from `x-office-document` to `applications-x-hwp` 

https://user-images.githubusercontent.com/54065734/156999697-dae78a7e-11a3-4b6b-a5e5-329f0395bb85.mp4

`

**Review by Muqtxdir — CHANGES_REQUESTED** [review]

**Comment by kuroehanako** [comment]
fixed problem.

**Comment by Muqtxdir** [comment]
soory for the confusion, I meant changing the baseplate title from `x-office-document` to `application-x-hwp` (the one in layers) not just the doc-name.

_ then you also need to render the mimetype:_
```bash
# inside yaru/icons/src/fullcolor
./render-bitmaps.py application-x-hwp
```

**Review by Muqtxdir — APPROVED** [review]

**Review by Jupi007 — APPROVED** [review]
LGFM :+1:

**Comment by Muqtxdir** [comment]
_@kuroehanako thanks for the nicely-done mimetype as a first-contribution towards yaru_
merging this !

**Pull request merged**